### ISSUE TITLE
Create Github Actions Pipeline

### ISSUE THREAD

**Issue by neotheprogramist** [neotheprogramist/cairo-webauthn#32]
The pipeline should run the following jobs:
- cargo fmt
- cargo clippy
- scarb fmt
- [typos](https://github.com/marketplace/actions/typos-action)
- cargo test
- scarb test

**Comment by glihm** [comment]
This would be great to run tests in this pipeline. :+1:

**Comment by neotheprogramist** [comment]
@glihm 
Yes I agree, so I think we can do tests first (moving from python to rust) and then the pipeline
